docs/topics/mta-web-review-tab.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-review-tab_{context}"]
+= Application review tab
+
+You can review the settings of any application or archetype settings directly from the list.
```

Comment:
Replace 1 with 2
1)You can review the settings of any application or archetype settings directly from the list.

2)You can view the review details of any application  on the application details page.

Reply:
Removed line 1 and replace it with line 2.
See https://github.com/windup/windup-documentation/pull/808